site/en/docs/web-platform/navigation-api/index.md

```diff
@@ -92,90 +108,252 @@ Links might come and go on your page, and they're not the only way users can nav
 E.g., they may submit a form or even use an [image map].
 Your page might deal with these, but there's a long tail of possibilities which could just be simplified—something that the new Navigation API achieves.
 
+Additionally, the above doesn't handle back/forward navigation. There's another event for that, `"popstate"`.
+
 Personally, the History API often _feels_ like it could go some way to help with these possibilities.
 However, it really only has two surface areas: responding if the user presses Back or Forward in their browser, plus pushing and replacing URLs.
 It doesn't have an analogy to "navigate", except if you manually set up listeners for, e.g., click events, as demonstrated above.
 
-## Transition
+## Deciding how to handle a navigation
 
-When your code calls `transitionWhile()` from within its "navigate" listener, it informs the browser that it's now preparing the page for the new, updated state; and that the navigation may take some time. The `Promise` you pass to `transitionWhile()` tells the browser how long the navigation takes.
+The `navigateEvent` contains a lot of information about the navigation that you can use to decide whether to intercept a particular navigation.
```

Comment:
```suggestion
The `navigateEvent` contains a lot of information about the navigation that you can use to decide how to deal with a particular navigation.
```

maybe? Since this is also about whether you want to cancel.